packages/web-app-external/src/App.vue

```diff
@@ -52,14 +68,98 @@ export default defineComponent({
     LoadingScreen
   },
   setup() {
+    const store = useStore()
+    const router = useRouter()
+    const currentRoute = useRoute()
+    const clientService = useClientService()
+    const { loadFileInfoByIdTask } = useLoadFileInfoById({ clientService })
     const appName = useRouteQuery('app')
     const applicationName = computed(() => queryItemAsString(unref(appName)))
+
+    const fileIdQueryItem = useRouteQuery('fileId')
+    const fileId = computed(() => {
+      return queryItemAsString(unref(fileIdQueryItem))
+    })
+
+    const driveAliasAndItem = useRouteParam('driveAliasAndItem')
+
+    const getMatchingSpace = (id): SpaceResource => {
+      return store.getters['runtime/spaces/spaces'].find((space) => id.startsWith(space.id))
+    }
+    const findMatchingMountPoint = (id: string | number): SpaceResource => {
```

Comment:
I'd love to have consistency here

**get**MatchingSpace
**find**MatchingMountPoint